test/suites/ACID.m

```diff
@@ -2379,6 +2380,40 @@
     set(h_t,'BackgroundColor','g');
 end
 % =========================================================================
+function [stat] = overlappingPlots()
+    stat.description = 'Overlapping plots with zoomed data and varying background.';
+    stat.issues = 6;
+
+    % create pseudo random data and convert it from matrix to vector
+    l = 16^2;
+    l_zoom = 64;
+    wave = magic(sqrt(l));
```

Comment:
You suggested in https://github.com/matlab2tikz/matlab2tikz/issues/414 to avoid `magic()`. 
Why don't we just use `sin()`?